scripts/webpack/removeFrontMatter.js

```diff
@@ -26,7 +26,18 @@ const matter = require('gray-matter');
 
 module.exports = src => {
   return new Promise(resolve => {
-    const { content } = matter(src);
-    return resolve(content);
+    const { content, data } = matter(src);
+
+    if (Object.keys(data).length === 0) {
+      return resolve(content);
+    }
+
+    // Exports must go at the bottom, as exports before imports aren't allowed.
+    const contentWithMetadataExport = `${content}
+
+export const metadata = ${JSON.stringify(data)};
+    `;
+
+    return resolve(contentWithMetadataExport);
```

Comment:
Had to reinstate this behaviour so that when we import mdx from non-autogenerated files we can access the metadata.